File: CITS_solution/CITS_WebServices/Models/Evaluation.cs
using System;
using System.Collections.Generic;

namespace CITS_WebServices.Models;

public partial class Evaluation
{
    public int EvaluationId { get; set; }

    public int CandidateId { get; set; }

    public int InterviewId { get; set; }

    public string? EvaluationType { get; set; }

    public double? Score { get; set; }

    public string? Feedback { get; set; }

    public DateTime? EvaluatedAt { get; set; }

    public virtual Candidate Candidate { get; set; } = null!;

    public virtual Interview Interview { get; set; } = null!;
}

File: CITS_solution/CITS_App/src/app/models/Evaluation/Evaluation.ts
// models/evaluation.model.ts
export interface Evaluation {
  evaluationId: number;
  candidateId: number;
  interviewId: number;
  evaluationType?: string | null;
  score: number | null;
  feedback: string | null;
  evaluatedAt?: Date | string | null; // use string if it comes as ISO string from API
}
/*  public int InterviewId { get; set; }

    public int CandidateId { get; set; }

    public DateTime ? ScheduledDateTime { get; set; }

    public string InterviewMode { get; set; } = null!;

    public int InterviewerId { get; set; }

    public int StageId { get; set; }

    public string Status { get; set; } = null!;*/
